client/my-sites/plan-features/summary.jsx

```diff
@@ -25,6 +25,21 @@ class PlanFeaturesSummary extends Component {
 		site: PropTypes.object.isRequired,
 	};
 
+	getTitle() {
```

Comment:
Minor, but it'd be nice to add a comment regarding why this isn't translated (bcs of AB test) so that we don't forget it when the AB test ends.

Reply:
@yoavf Good catch! I will add the comment. Thanks for the review. :)